dbt/models/pinval/schema.yml

```diff
@@ -126,7 +127,7 @@ models:
                 # There are a handful of known PINs that had the wrong class
                 # at modeling time, and so got a model value even though it
                 # should not have
-                error_if: ">29"
+                error_if: ">28"
```

Comment:
Previously this test was capturing the 2024 PIN that we are now manually setting to `is_report_eligible = false` (see above), so now we can drop the threshold to reflect that that particular PIN has been manually fixed.